### ISSUE TITLE
Availability on OpenWRT/LEDE

### ISSUE THREAD

**Issue by diizzyy** [sipcapture/sipgrep#29]
If you want to include, it's also available as a package on OpenWRT/LEDE

https://github.com/openwrt/packages/tree/master/net/sipgrep

Best regards,
Daniel

**Comment by lmangani** [comment]
Hi @diizzyy shall we remove the reference?

**Comment by diizzyy** [comment]
No, it's still available.  I just didn't see the need to keep this open.
